src/review.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="description" content="Mini Project to test out technologies for the Isuroon project." />
+        <meta name="keywords" content="Isuroon, Health, Review, Doctors" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="author" content="Hack4Impact Carleton" />
+        <title>Isuroon MiniProj 1</title>
+        <link rel="stylesheet" href="main.css" />
+        <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    </head>
+    <body>
+        <div id="review_container">
+            <span><a href="results.html" class="glyphicon glyphicon-chevron-left" id="back_button"></a></span>
+            <h1 id="doctor"></h1>
+            <!-- Insert review cards using script-->
+        </div>
+
+        <script src="review.js"> </script>
+        <script>
+            window.onload = function(){
+                var doctors_name = localStorage.getItem("doctors_name");
+                localStorage.removeItem('doctors_name');
+                document.getElementById('doctor').innerHTML = doctors_name;
+                createReviewCards();
+            };

```

Comment:
We may want to use POST requests (ex: www.something.com?doctor=Mark&zipcode=123456) to handle data forwarding so that people can bookmark a doctor's review page. Check out the [notes page](https://www.notion.so/h4i/Mini-Project-Notes-fea24e25201c47deb5e341f7cd859128) where I give an example with a custom anchor.

Reply:
Fixed! See commit [bb5e55e](https://github.com/h4icarleton/Isuroon-MiniProj-1/commit/bb5e55e31aada743085a5a2db247d737c0de3f37).